apps/sv/frontend/src/utils/types.ts

```diff
@@ -18,7 +18,8 @@ export interface UnfeatureAppProposal {
 
 export interface ConfigChange {
```

Comment:
As mentioned above, we use the ConfigChange to track changes made for dso/amulet config. It helps us display something like 

<img width="930" height="427" alt="Image" src="https://github.com/user-attachments/assets/0bc07d17-651d-495d-b435-64b2ea60ce99" />

It's also reused in this PR for proposal creation but since we need to submit json, an encoder will be needed to turn a list of config changes from the form into json.